third_party/nvfuser/csrc/fusion_segmenter.cpp

```diff
@@ -2647,15 +2642,16 @@ SegmentCandidateFinder::SegmentCandidateFinder(
   findSegments();
 }
 
-void SegmentCandidateFinder::findSegments() {
-  FUSER_PERF_SCOPE("Finding valid fusion segment solutions");
+void SegmentCandidateFinder::buildInitialSegments() {
```

Comment:
Factored out the part to create the initial graph so that it can be redone if necessary